### PR TITLE
Small tweak in chapter sorting of the example gallery

### PR DESCRIPTION
**Pull request by kolibril13**
Small resorting in the gallery.
Before:
image
After:
https://output.circle-artifacts.com/output/job/cbdbb18c-5feb-4fc4-b7b9-4c857e08a01f/artifacts/0/doc/build/html/gallery/index.html#
image

### CODE REVIEW

**Comment by jklymak** [comment]
This is an example where the pre-commit hook catches something that flake8 does not.  It would be nice if they were consistent.  @kolibril13 this needs some whitespace fixes....

**Comment by ksunden** [comment]
@jklymak Flake8 will never get this error, as flake8 is _only_ about python source files.
Each pre-commit hook can be configured to run on specific files (the trailing whitespace one runs on all text files except SVGs)

This particular trailing whitespace is in a README.txt.

If it were in python source, it would be `W291` (from pycodestyle), which is selected by our flake8 config.

File: examples/axes_grid1/README.txt [hunk 1]
```diff
@@ -2,5 +2,5 @@
 
 .. _axes_grid1-examples-index:
 
-The axes_grid1 module
-=====================
+Module - axes_grid1
```
Comment (ksunden) [inline examples/axes_grid1/README.txt:5]:
```suggestion
Module - axes_grid1
```

File: examples/axisartist/README.txt [hunk 2]
```diff
@@ -2,5 +2,5 @@
 
 .. _axisartist-examples-index:
 
-The axisartist module
-=====================
+Module - axisartist
```
Comment (ksunden) [inline examples/axisartist/README.txt:5]:
```suggestion
Module - axisartist
```

**Review by tacaswell — APPROVED** [review]

**Comment by tacaswell** [comment]
Even though it is a documentation change, I'm going to err on the side of not backporting.

**Pull request merged**